app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -93,8 +93,6 @@
   }
 }
 
-
-
```

Comment:
Could this (and other similar linting commits) be merged with an appropriate commit? I wouldn't normally ask but there are a lot of commits in this PR.

Reply:
Still in the process of tidying up the history on this branch but will mark this as resolved as I've already squashed the particular commit this comment was about.